packages/kit/assets/components/error.svelte

```diff
@@ -1,3 +1,11 @@
+<script context="module">
+	export async function load({ error, status }) {
```

Comment:
I'm just going to go ahead and change so we can merge this
```suggestion
	export function load({ error, status }) {
```